app/controllers/HomepageController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace Application\Controllers\HomepageController;
+
+use Application\Controllers\Controller;
+
+require_once(ROOT . '/app/controllers/controller.php'); //
```

Comment:
Normally you don't need that. check how to remove the `require` and use the `use` instead

Reply:
Yes a know but here the use is recognized by VS code but it's not working, for now i don't know why????


Fatal error: Uncaught Error: Class "Application\Controllers\Controller" not found in /Users/Tanguy/Documents/sites/blog-omega/app/controllers/HomepageController.php:11 Stack trace: #0 /Users/Tanguy/Documents/sites/blog-omega/public/index.php(13): require_once() #1 {main} thrown in /Users/Tanguy/Documents/sites/blog-omega/app/controllers/HomepageController.php on line 1
<img width="660" alt="Capture d’écran 2022-07-05 à 08 30 36" src="https://user-images.githubusercontent.com/18627668/177264384-3ddca659-2bd8-4a41-ae4c-820cc4cfbb44.png">
<img width="660" alt="Capture d’écran 2022-07-05 à 08 30 36" src="https://user-images.githubusercontent.com/18627668/177264482-ef164df9-faae-40dc-adb6-5a3bdd49926d.png">
1

Reply:
It's ok know by coding a special autoloading which fix this isuue